packaging/language/maven_artifact.py

```diff
@@ -26,6 +26,8 @@
 import hashlib
 import sys
 import posixpath
+from ansible.module_utils.basic import *
+from ansible.module_utils.urls import *
```

Comment:
I think the two lines above should stay at the [bottom of the file](http://docs.ansible.com/ansible/developing_modules.html#common-module-boilerplate). There is a reason for that but I forgot it ;o)